app/services/hmpps_api/prison_api/offender_api.rb

```diff
@@ -40,6 +41,7 @@ def self.list(prison, page = 0, page_size: 20)
           search_payload = search_data.fetch(offender_no)
           HmppsApi::OffenderSummary.new(payload,
                                         search_payload,
+                                        category: offender_categories[offender_no],
```

Comment:
this allows nils - think it should be a fetch? If we're worried about inconsistent caches again, then offender doesn't exist if they don't have a category?

Reply:
As above: there might not be an active category assessment for every offender, so sometimes this can be `nil` (and that's expected behaviour).